.editorconfig

```diff
@@ -0,0 +1,557 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
```

Comment:
Could you flip the setting to insert a new line at the end of files?

I gave up on tabs VS spaces long ago but this one I still care about.

Reply:
Done.

Reply:
Oh, I guess it does nothing for Java/Kotlin source files. Unfortunate.